engine/src/main/java/com/datatorrent/stram/client/DTConfiguration.java

```diff
@@ -73,7 +74,9 @@
   public static class ValueEntry
```

Comment:
Please make this private. Introduce PropertyInfo class in AppPackage, with value and description.

You can populate the description field from DtConfiguration.getDescription method